dashboard/app/models/pd/async_workshop_handler.rb

```diff
@@ -1,3 +1,5 @@
+require "#{Rails.root}/lib/sqs/sqs_queue"
```

Comment:
i think you can do `require 'sqs/sqs_queue'` here rather than including the full path